test/server.spec.js

```diff
@@ -2,6 +2,10 @@ const { initServer } = require("../src/server");
 const axios = require("axios");
 
 describe("server", () => {
+  afterEach(() => {    
```

Comment:
Why do we need this?

Reply:
Without it, we are not undoing the mocks, because by default jest doesn't do it.